C/CFITSIO/build_tarballs.jl

```diff
@@ -7,34 +7,40 @@ version = v"3.47.0"
 
 # Collection of sources required to build CFITSIO
 sources = [
-    "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-3.47.tar.gz" =>
+    "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-$(version.major).$(version.minor).tar.gz" =>
     "418516f10ee1e0f1b520926eeca6b77ce639bed88804c7c545e74f26b3edf4ef",
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/cfitsio*
-if [[ "${target}" == *freebsd* ]]; then
-    CC=/opt/${target}/bin/${target}-gcc
-    LD=/opt/${target}/bin/${target}-ld
-fi
-./configure --prefix=$prefix --host=$target --enable-reentrant
+
+# Explanation of `CURLCONFIG`: CFITSIO uses the application `curl-config` to
+# check configuration of curl, needless to say tha this is perfectly useless and
+# dangerous during a cross-compilation, thus we need to use `pkg-config` instead
+./configure --prefix=$prefix --host=$target --enable-reentrant \
+    CURLCONFIG="$(pkg-config --cflags --libs libcurl libssh2 | sed "s?/workspace/destdir/lib?${libdir}?")"
```

Comment:
I had to manually change `/workspace/destdir/lib` -> `/workspace/destdir/bin`.  I think this is a problem with all pkg-config files on Windows?